packages/amplify-e2e-tests/src/__tests__/pull-2.test.ts

```diff
@@ -0,0 +1,60 @@
+import { addAuthIdentityPoolAndUserPoolWithOAuth } from '../../../amplify-e2e-core/src/categories/auth';
+/* eslint-disable spellcheck/spell-checker */
+import {
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  getAppId,
+  initJSProjectWithProfile,
+  pullProject,
+} from '@aws-amplify/amplify-e2e-core';
+import { createIDPAndUserPoolWithOAuthSettings, getShortId, importIdentityPoolAndUserPool } from '../import-helpers';
+import { AmplifyFrontend } from 'amplify-cli-core';
```

Comment:
Does this import need to be updated?

```suggestion
import { AmplifyFrontend } from '@aws-amplify/amplify-cli-core';
```